doc/build/faq/connections.rst

```diff
@@ -396,8 +396,8 @@ If using a SQLite ``:memory:`` database, or a version of SQLAlchemy prior
 to version 0.7, the default connection pool is the :class:`.SingletonThreadPool`,
 which maintains exactly one SQLite connection per thread.  So two
 connections in use in the same thread will actually be the same SQLite
-connection.   Make sure you're not using a :memory: database and
-use :class:`.NullPool`, which is the default for non-memory databases in
+connection.  Make sure you're not using a :memory: database which will
+use :class:`.QueuePool`, the default for non-memory databases in
```

Comment:
> Make sure you're not using a :memory: database which will
use :class:`.QueuePool`

this is not correct.

Maybe something like
> Make sure you're not using a :memory: database, so that the egine will
use :class:`.QueuePool`

Reply:
Understood. Fixing...

Reply:
I was probably drunk when I wrote that 😄 . Fixed.